package.json

```diff
@@ -71,6 +73,7 @@
     "husky:pre-commit": "run-p spellcheck husky:lint-staged",
     "lint": "eslint --resolve-plugins-relative-to \"./node_modules/@ridedott/eslint-config\" \"**/*.ts\"",
     "lint:fix": "eslint --fix --resolve-plugins-relative-to \"./node_modules/@ridedott/eslint-config\" \"**/*.ts\"",
+    "prepublishOnly": "npm run build",
```

Comment:
Why would you use this hook if it's specified in the CD config already?

Reply:
removed with https://github.com/ridedott/eslint-plugin/pull/30/commits/0d1b41b2e6fa7c18ddca54619c872d55e5350a02